governance/default.go

```diff
@@ -958,7 +961,8 @@ func (gov *Governance) VerifyGovernance(received []byte) error {
 	}
 
 	for k, v := range rChangeSet {
-		if GovernanceKeyMap[k] == params.GoverningNode {
+		if GovernanceKeyMap[k] == params.GoverningNode ||
```

Comment:
You didn't import GovParamContract in `GetGovernanceItemsFromChainConfig`. Can I ask the reason? And, you didn't import in `HandleGovernanceVote`.. but the `GovParamContract` is set by header voting, right? Is it not necessary..??

Reply:
@yoomee1313 

1. About `GetGovernanceItemsFromChainConfig()`
`GetGovernanceItemsFromChainConfig()` is used to write the genesis block.
Thus, changing the implementation of `GetGovernanceItemsFromChainConfig()` will break backward compatibility. That is, running `kcn init` with kcn v1.9 first, and then running `kcn init` with kcn v1.10 will produce different genesis block, which will lead to fail.
This will be addressed in the upcoming bugfix PR.

2. About `HandleGovernanceVote()`
AFAIK `HandleGovernanceVote()` is for special votes such as AddValidator or GoverningNode. GoverningNode is there to ensure that the new governing node is in the validator set. It is not necessary to change `HandleGovernanceVote()`. The vote will show like this:
```
> governance.vote("governance.govparamcontract", "0x1234")
Error: Your vote couldn't be placed. Please check your vote's key and value
    at web3.js:3278:20
    at web3.js:6810:15
    at web3.js:5221:36
    at <anonymous>:1:1
> governance.vote("governance.govparamcontract", "0x0000000000000000000000000000000000000000")
"Your vote is prepared. It will be put into the block header or applied when your node generates a block as a proposer. Note that your vote may be duplicate."

... a few blocks later

> governance.itemsAt(500)["governance.govparamcontract"]
"0x0000000000000000000000000000000000000000"
```

Reply:
@ian0371 thanks for your answer! It helps a lot.